components/applications-service/integration_test/global_test.go

```diff
@@ -129,24 +193,70 @@ func assertServiceGroupsEqual(t *testing.T, expected, actual *applications.Servi
 //   "unknown": 1
 // }
 // TODO @afiune add more apps & envs
-func habServicesMatrix() []*applications.HabService {
-	return []*applications.HabService{
+func habServicesMatrix() []*habitat.HealthCheckEvent {
+	return []*habitat.HealthCheckEvent{
 		// service_group 1 <-> With a Health Status = 'OK'
-		NewHabServiceMsg("sup1", a, e, "default", "core", "redis", "0.1.0", "20190101121212", "OK", "stable", "test"),
-		NewHabServiceMsg("sup2", a, e, "default", "core", "redis", "0.1.0", "20190101121212", "OK", "stable", "test"),
-		NewHabServiceMsg("sup3", a, e, "default", "core", "redis", "0.1.0", "20190101121212", "OK", "stable", "test"),
+		NewHabitatEvent([]MessageOverrides{
```

Comment:
can't we pass the arguments instead of making a slice and "splatting" it?

Reply:
How so? Like it was before? I feel this is a better way but I would like to hear why and how would you like this to be updated! Looove any feedback! 💯

Reply:
I could change the function argument from:
```go
func NewHabitatEvent(overrides ...MessageOverrides) *habitat.HealthCheckEvent {
```
To:
```go
func NewHabitatEvent(overrides []MessageOverrides) *habitat.HealthCheckEvent {
```

Would that work @danielcbright ?